### PR TITLE
Add workspace templates to extended tests and update base template versions

### LINKED ISSUE #3702: Add Unrestricted and airlock review workspaces to e2e tests

### PR DESCRIPTION
**Pull request by marrobi**
# Resolves #3702

## What is being addressed
Add workspace creation tests to extended tests.

### CODE REVIEW

**Comment by marrobi** [comment]
Tests are now failing correctly - https://github.com/microsoft/AzureTRE/actions/runs/6103122391

Wil fix the bundles.

**Comment by marrobi** [comment]
/test-force-approve

passed - https://github.com/microsoft/AzureTRE/actions/runs/6107126915

File: e2e_tests/test_workspace_templates.py [hunk 1]
```diff
@@ -36,3 +38,20 @@ async def test_get_workspace_template(template_name, verify) -> None:
     admin_token = await get_admin_token(verify)
     async with get_template(template_name, strings.API_WORKSPACE_TEMPLATES, admin_token, verify) as response:
         assert_status(response, [status.HTTP_200_OK], f"Failed to GET template: {template_name}")
+
+
+@pytest.mark.extended
+@pytest.mark.parametrize("template_name", workspace_templates)
```
Comment (tamirkamara) [inline e2e_tests/test_workspace_templates.py:44]: workspace_templates include the base one too which is anyway checked as part of the guacamole tests. I think you should somehow split it how to save 30 mins of tests.

Comment (marrobi) [reply e2e_tests/test_workspace_templates.py]: Is that any better?

**Pull request merged**